scripts/packages_exporter.py

```diff
@@ -18,9 +21,12 @@
 from alws import database
 from alws import models
 from alws.config import settings
-from alws.utils.pulp_client import PulpClient
+from alws.routers.sign_key import get_sign_keys
```

Comment:
Please do not use fastapi things in console scripts

Reply:
Fixed